packages/gatsby/src/redux/nodes.ts

```diff
@@ -133,60 +121,52 @@ const getResolvedNode = (typeName, id) => {
   return node
 }
 
-exports.getResolvedNode = getResolvedNode
-
-const addResolvedNodes = (typeName, arr) => {
+export const addResolvedNodes = (typeName: string): INode[] => {
+  const resolvedNodes: INode[] = []
   const { nodesByType, resolvedNodesCache } = store.getState()
-  const nodes /*: Map<mixed> */ = nodesByType.get(typeName)
+  const nodes = nodesByType.get(typeName)
 
   if (!nodes) {
-    return
+    return resolvedNodes
```

Comment:
This is a small API change but I'm sure the call sites have been updated accordingly (TS would complain, that's probably what lead to this change in the first place).

I wonder if it would be better / clearer to return `[]` here, than to refer to the existing var. But not super important.

Reply:
I have verified that this is correct. I can return an empty array. I like that for future maintenance potential issues.